web/crux/templates/dyrector.io.json

```diff
@@ -0,0 +1,407 @@
+{
+  "name": "dyrector.io",
+  "description": "dyrector.io is a self-hosted container management platform.\nINFORMATION:\n1) If you wish to run this software without reCAPTCHA, set DISABLE_RECAPTCHA to true, and remove the RECAPTCHA related secrets from the frontend(crux-ui) application.",
```

Comment:
This should contain some basic guidelines how to fill those envs as I wrote in the other comment.

Reply:
Add "Link the docs"